### ISSUE TITLE
Provide better messaging to user when database file is not found on startup

### ISSUE THREAD

**Issue by wedega** [keepassxreboot/keepassxc#8730]
latest v2.7.4
I want to open the last database when the program starts.
That does not work. Only when I choose Open Database in the menu and re-enter the whole long path does it work.

https://kee.domain.de/nextcloud/remote.php/.....-
in the program for recently used databases is then
\\kee.domain.de@SSL\DavWWWRoot\nextcloud\.....
I think the @SSL\DavWWWRoot in the path doesn't work.
If I manually enter the original path in the program when opening the database, everything works perfectly

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )

## Context
[NOTE]: # ( Give us any additional information you may have. )

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - VERSION
Revision: REVISION

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows/Linux/macOS
Desktop Env: Gnome/KDE/XFCE/Mate/Cinnamon
Windowing System: X11/Wayland

**Comment by droidmonkey** [comment]
Do you have your ~/.cache directory RAM mounted or cleared on logoff?

**Comment by wedega** [comment]
Hello, first of all thanks for the answer. 
No cache cleared.
When the PC wakes up from sleep and I want to use the program, the DB cannot be opened. It can only be used again if I click on open database and enter the long path manually. Unable to open last DB used.

**Comment by droidmonkey** [comment]
What is possibly happening is that the location is not mounted until you go through the file browser which kicks off a mount effort. Since this is a "live" folder that can cause problems. We don't recommend accessing your database this way, but instead have nextcloud sync files locally and you access the local cache. 

If you close keepassxc and reopen it (without reboot or sleep) , does the database open just fine?

**Comment by wedega** [comment]
Ok, does that mean it is not good or not possible to access a DB on the web space?
If you close keepassxc and reopen it (without reboot or sleep) , does the database open just fine?
Yes it do so

**Comment by wedega** [comment]
I think I got it now. Without turning off the PC or sending it to sleep, everything is fine. After rebooting or hibernating etc. I have to re-enter the path.

**Comment by wedega** [comment]
Jonathan thanks a lot for the effort and the replies.
All the best

**Comment by droidmonkey** [comment]
Yes you will need to setup an automount of that folder when your computer wakes from sleep or boot up. Behind the scenes, we attempt to open the file but if Qt returns that it doesn't exist then we cancel and close it. 

I do think there should be a better ux here that explains what happened and perhaps even keeps the database tab open with a not found message or similar. I am not sure how we could trigger a remount from our application since that is being handled by Gnome.

**Comment by wedega** [comment]
> Yes you will need to setup an automount of that folder when your computer wakes from sleep or boot up. Behind the scenes, we attempt to open the file but if Qt returns that it doesn't exist then we cancel and close it.

Ahh mmm how can i do it? 
I'm not a developer, more a newbie

**Comment by droidmonkey** [comment]
Not sure there are a lot of ways to do it and depends on how you setup the interface.  You'll have to scour the support forums for your setup.

**Comment by wedega** [comment]
OK thank you
